packages/core/demo/index.ts

```diff
@@ -1,10 +1,10 @@
 import {
 	BarChart,
+	ScatterChart,
 	LineChart,
 	PieChart,
 	DonutChart,
-	ComboChart,
-	ScatterChart
+	ComboChart
```

Comment:
Why are we moving these imports around?

Reply:
I just moved combo to the bottom as its sub-charts include the charts above and, scatter before line. It doesn't really matter, you want me to revert this?

Reply:
I see. We can keep it, it doesn't really matter